src/users/users.controller.ts

```diff
@@ -62,7 +65,7 @@ export class UsersController {
   @Put(':id')
   @Roles([UserRoleEnum.ADMIN, UserRoleEnum.USER])
   update(
-    @Param('id', ParseUUIDPipe) id: string,
```

Comment:
If I understand correctly, I think this changes `PUT /users/:id` into `PUT /users/:id?id=:id`. Let me know if I misunderstood something!